src/main/java/de/zalando/tip/zalenium/proxy/DockerSeleniumRemoteProxy.java

```diff
@@ -261,10 +261,12 @@ void copyVideos(final String containerId) throws IOException, DockerException, I
                     continue;
                 }
                 String fileExtension = entry.getName().substring(entry.getName().lastIndexOf('.'));
+                String folderName = entry.getName().substring(0, entry.getName().indexOf("/") + 1);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Put single-quotes around '/' to use the faster "indexOf(char)" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3027)